Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -745,6 +747,37 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     }
 }
 
+//MARK: Universal Bridge support
+extension MiniAppScriptMessageHandler {
+    func sendJsonToHostApp(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.jsonInfo else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: MiniAppJavaScriptError.invalidPermissionType.rawValue)
```

Comment:
We don't need any permissions here. So throwing a permission related error here is not required.
Please use 'prepareMAJavascriptError(MiniAppJavaScriptError.unexpectedMessageFormat)' or any other error

Reply:
For this I have now created new error enum object `UniversalBridgeError` to cater for universal bridge related errors. With this I can now use the `prepareMAJavascriptError(error)`

Reply:
@GondiTilak . No, we don't need any custom permissions to use this interface. Throwing an error related to invalid permission is misleading.

Reply:
This is now updated to use the `MiniAppJavaScriptError`